atd-vze/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "atd-vz-data",
-  "version": "1.25.0",
+  "version": "1.27.1",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "atd-vz-data",
-      "version": "1.25.0",
+      "version": "1.27.1",
```

Comment:
out of curiosity—are y'all skipping versions?

Reply:
I've found that if no one has npm-installed between versions, the package-lock won't update and so its version in github doesn't match the package.json